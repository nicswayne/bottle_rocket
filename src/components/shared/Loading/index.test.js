import React from 'react';
import { shallow, render } from 'enzyme';

import Loading from '.';

it('Loading component should render without crashing.', () => {
    shallow(<Loading />);
});

it('Loading component should match snapshot.', () => {
    const wrapper = render(<Loading />);

    expect(wrapper).toMatchSnapshot();
});
