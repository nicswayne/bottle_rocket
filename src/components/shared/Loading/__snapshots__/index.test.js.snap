// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should match snapshot. 1`] = `
<div
  class="loading"
>
  <h2>
    Loading...
  </h2>
</div>
`;
